src/config/version1.mlh

```diff
@@ -0,0 +1,50 @@
+[%%define ledger_depth 35]
+[%%import "/src/config/curve/medium.mlh"]
```

Comment:
Why is this needed if we are updating the txn version using sql? 
If it is required then is this config to be used only for generating the migration app for migrating mainnet DB and not for the daemon builds? If so, I'd suggest naming it appropriately- maybe `berkeley_archive_migration` and creating a separate build job for the app.  Otherwise this could cause some confusion

Reply:
This is only needed to for the replayer. I would rename it as you suggested.